sdks/python/README.md

```diff
@@ -467,33 +354,16 @@ SAMPLE_DATA = [('a', 1), ('b', 10), ('a', 2), ('a', 3), ('b', 20)]
 p.run()
 ```
 
-The `google/cloud/dataflow/examples/cookbook/combiners_test.py` file in the
-source distribution contains more combiner examples.
-
-### More Examples
-
-The `google/cloud/dataflow/examples` subdirectory in the
-source distribution has some larger examples.
+The [combiners_test.py](https://github.com/apache/incubator-beam/blob/python-sdk/sdks/python/apache_beam/examples/cookbook/combiners_test.py) file contains more combiner examples.
 
 ## Organizing Your Code
 
-Many projects will grow to multiple source code files. It is beneficial to
-organize the project so that all the code involved in running a workflow can be
-built as a Python package so that it can be installed in the VM workers
-executing a job.
+Many projects will grow to multiple source code files. It is recommended that you organize your project so that all code involved in running your pipeline can be built as a Python package. This way, the package can easily be installed in the VM workers executing the job.
 
-Please follow the example in `google/cloud/dataflow/examples/complete/juliaset`.
-If the code is organized in this fashion then you can use the `--setup_file`
-command line option to create a source distribution out of the project files,
-stage the resulting tarball and later install it in the workers executing the
-job.
+Please follow the example in `google/cloud/dataflow/examples/complete/juliaset`. If the code is organized in this fashion, you can use the `--setup_file` command line option to create a source distribution out of the project files, stage the resulting tarball, and later install it in the workers executing the job.
 
 ## Contact Us
```

Comment:
Contact is the wrong tone. Perhaps just 'More information'.

And couldn't hurt to have a call to action to come contribute.